src/vivarium_gates_nutrition_optimization/data/loader.py

```diff
@@ -145,7 +150,43 @@ def _load_em_from_meid(location, meid, measure):
     return vi_utils.sort_hierarchical_data(data).droplevel("location")
 
 
-# TODO - add project-specific data functions here
+##################
+# Pregnancy Data #
+##################
+
+
+def get_pregnancy_end_rate(location: str) -> pd.DataFrame:
+    asfr = get_data(data_keys.PREGNANCY.ASFR, location)
+    sbr = get_data(data_keys.PREGNANCY.SBR, location)
+    sbr = sbr.reset_index(level="year_end", drop=True).reindex(
+        asfr.index, level="year_start", fill_value=0.0
```

Comment:
Should probably confirm that filling w/ 0 is appropriate. 

Can you explain what this is doing? I'm assuming this reindexing is so the `.multiply` below works but I'm confused why you reset to 'year_end' and then reindex to asfr's `year_start`

Reply:
pandas gets mad about trying to reindex based on a level if both objects are `MultiIndex`. The `reset_index` just drops the `year_end` level from `sbr`, so that there is a single unambiguous level upon which to join the dataframes. Since `year_end` is uniquely specified by `year_start` in this case, it is in some sense extraneous (and the reindexing adds it back in).

I'm not sure if this is the best way to do this, but it is done the same way several times in IV Iron 
(cf `loader.py` ln 203, 529, 590), so I just followed suit.

Reply:
probably the fillna isn't needed though